src/main/java/use_cases/CSVDatabase.java

```diff
@@ -0,0 +1,56 @@
+package use_cases;
```

Comment:
I'd change this package to be something like "drivers" or "database" , something like that, but other than that all is good.

Reply:
agreed

Reply:
That makes sense, I'll make that change.